src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -204,6 +208,86 @@ public void writeVariableExpr_scopedStaticFinalDecl() {
     assertEquals(writerVisitor.write(), "public static final boolean x");
   }
 
+  @Test  
+  public void writeNormalBlockComment() {
```

Comment:
Could you please add a test with badly-formatted input?

Reply:
sure, technically comment should not have `*/` in the middle which will break the formatter. And also confirmed we do not have such comment in baseline, so use this as a bad test case.

Reply:
done.